cartography/intel/azure/compute.py

```diff
@@ -43,6 +43,7 @@ def get_vm_list(credentials: Credentials, subscription_id: str) -> List[Dict]:
         for vm in vm_list:
             x = vm['id'].split('/')
             vm['resource_group'] = x[x.index('resourceGroups') + 1]
+            vm['requireGuestProvisionSignal'] = vm.get('properties', {}).get('osProfile', {}).get('require_guest_provision_signal', True)
```

Comment:
should this be true or false when it's null. Is there a documentation which shows this logic?